app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,18 @@
+class SessionsController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:create]
+
+  def create
+    session.update(
+      current_user: User.build_from(auth_hash),
```

Comment:
Generally, it's not good practice to store activerecord objects in session.. I know we're using 'build_from' rather than using a db-persisted object... but I'm still not sure that this is a good idea. If the user gets to modify the session somehow, they can introduce any value they want into that object.

We should instead store the uuid of the user in the session, and create a method that will fetch the user for that uuid and make it available for other code.

Something along the lines of:
```
session.update(current_user_uuid: .... )
def session_user
  User.find_by_uuid(session.current_user_uuid)
end
```